src/webgpu/shader/execution/robust_access_vertex.spec.ts

```diff
@@ -0,0 +1,40 @@
+export const description = `
+Test vertex attributes behave correctly (no crash / data leak) when accessed out of bounds
+`;
+
+/*
```

Comment:
I think let's start out putting the test plan inside the description string like in #227. We can change it later if we decide it's unwieldy. Format doesn't need to be consistent (but could be consistent with how I've been writing, if you think that's a good format https://hackmd.io/@kainino0x/H1MwoqqAU)

Reply:
Moved it in, seems like the hackmd link is down?

Reply:
Sorry, link moved and I guess it doesn't redirect: https://hackmd.io/@webgpu/H1MwoqqAU